plotly/tests/test_optional/test_figure_factory.py

```diff
@@ -1267,6 +1267,15 @@ def test_scatter_plot_matrix_kwargs(self):
                                exp_scatter_plot_matrix['layout'])
 
 
+class TestGantt(NumpyTestUtilsMixin, TestCase):
+
+    def test_df_dataframe(self):
+
+        # validate dataframe has correct column names
+        df1 = pd.DataFrame([[2, 'Apple']], columns=['Numbers', 'Fruit'])
+        self.assertRaises(PlotlyError, tls.FigureFactory.create_gantt, df1)
```

Comment:
Not thorough necessarily, just an `all_args`. I just want to make sure that using a dataframe will continue to work.